lib/profiles/dosomething/drupal-org.make

```diff
@@ -275,6 +275,10 @@ libraries[predis][download][url] = "https://github.com/nrk/predis.git"
 ; Pin to 0.8 release, which still has the lib directory.
 libraries[predis][download][revision] = "4123fcd85d61354c6c9900db76c9597dbd129bf6"
 
+; Stathat
+libraries[stathat][download][type] = "get"
```

Comment:
Is `get` a valid type? Did you mean `file`?
http://cgit.drupalcode.org/drush_make/plain/README.txt?id=refs/heads/6.x-2.x